recipes/lora_ppo_single_device.py

```diff
@@ -30,17 +30,9 @@
     validate_state_dict_for_lora,
 )
 from torchtune.recipe_interfaces import FTRecipeInterface
+from torchtune.utils import ppo as ppo_utils
```

Comment:
perhaps ppo_utils should be a standalone module under the `torchtune` namespace? or just under the `torchtune.utils.ppo_utils` namespace so we have either from `torchtune import ppo_utils`, or `from torchtune.utils import ppo_utils`.

Reply:
going to go with the latter.